BannedApiAnalyzer/BannedApiAnalyzer.CSharp/ApiDesignRules/CSharpSymbolIsBannedAnalyzer.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
```

Comment:
Just want to call out the different of copyright and license info. I'm not a lawyer, would this be a problem?

```
// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Reply:
The attribution for the original content is here:

https://github.com/DotNetAnalyzers/BannedApiAnalyzer/blob/a6a45026fb5ea6121deac6e51b329cbe2c3181ac/THIRD-PARTY-NOTICES.txt#L10-L24